### ISSUE TITLE
Support for MySQL

### ISSUE THREAD

**Issue by yerke** [oxidizer-rs/oxidizer#11]
Hello.

Do you plan to work on supporting on other DB engines except for PostgreSQL? I would love to see support for MySQL specifically.

Thanks.

**Comment by GustavoKatel** [comment]
Yes, this is planned. I will update the readme with planned features in the new few days.
